opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -208,4 +208,11 @@ message LogRecord {
   //   - the field is not present,
   //   - the field contains an invalid value.
   bytes span_id = 10;
+
+  // A globally unique identifier of event category/type among all events. All
+  // events with the same event_name MUST conform to the same schema for both their
+  // attributes and their body.
```

Comment:
Consider putting some limit here (e.g. event name should have <= 128 characters, and should only use Unicode BMP)

Reply:
Added recommendation to be short and <= 256 chars.

We never specify character set in proto so I'd prefer not to add it. On semconv side we have and enforce stricter limits.